resource/resource.go

```diff
@@ -31,13 +31,14 @@ const (
 	envVarTags = "OC_RESOURCE_TAGS"
 )
 
-// Resource describes an entity about which data is exposed.
+// Resource describes an entity about which identifying information and metadata is exposed.
```

Comment:
It's also the target for metrics to be recorded against, no? Isn't that the primary purpose?